HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -1,9 +1,22 @@
 CLASS::SequenceableCollection
-summary::Abstract superclass of integer indexable collections
+summary::Abstract superclass of indexable collections
 categories::Collections>Ordered
 
 DESCRIPTION::
-SequenceableCollection is a subclass of Collection whose elements can be indexed by an Integer. It has many useful subclasses; link::Classes/Array:: and link::Classes/List:: are amongst the most commonly used.
+link::Classes/SequenceableCollection:: is a subclass of link::Classes/Collection:: whose items can be indexed by a link::Classes/SimpleNumber::. It has many useful subclasses; link::Classes/Array:: and link::Classes/List:: are amongst the most commonly used.
+
+subsection::Index
```

Comment:
Perhaps this subsection could be called "Indexing"

Reply:
Thanks! Done in https://github.com/supercollider/supercollider/pull/6536/commits/027b55d840163edc880635a823f21bb4652aa5ab.